packages/mobile/src/web3/saga.ts

```diff
@@ -48,40 +49,45 @@ const MNEMONIC_BIT_LENGTH = (ETH_PRIVATE_KEY_LENGTH * 8) / 2
 
 const TAG = 'web3/saga'
 // The timeout for web3 to complete syncing and the latestBlock to be > 0
-export const SYNC_TIMEOUT = 60000
+export const SYNC_TIMEOUT = 3 * 60 * 1000 // 3 minutes
 const BLOCK_CHAIN_CORRUPTION_ERROR = "Error: CONNECTION ERROR: Couldn't connect to node on IPC."
 
 // checks if web3 claims it is currently syncing and attempts to wait for it to complete
 export function* checkWeb3SyncProgress() {
-  const zeroSyncMode = yield select(zeroSyncSelector)
-  if (zeroSyncMode) {
-    // In this mode, the check seems to fail with
-    // web3/saga/checking web3 sync progress: Error: Invalid JSON RPC response: "":
-    return true
-  }
```

Comment:
We want to run the loop now in zeroSyncMode too?

Reply:
Yes